nikola/plugin_categories.py

```diff
@@ -250,6 +250,7 @@ class PageCompiler(BasePlugin):
     friendly_name = ''
     demote_headers = False
     supports_onefile = True
+    use_dep_file = False  # If set to true, the .dep file is always written and added as a target
```

Comment:
Or perhaps we could make it so `.dep` always means file dependencies separated by newlines, and the WordPress compiler creates its custom `.wpdep` format?

And I think the default should be `True`.